spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -447,6 +447,22 @@
       end
     end
 
+    context 'with a user that accepted the rules of use more than 6 years ago' do
+      let(:rules_of_use_updated_at) { 7.years.ago }
```

Comment:
this value for the config store should be a plain int, right? Since we call the `.year.ago` inside the method

```suggestion
      let(:rules_of_use_updated_at) { 7 }
```

Reply:
whoops, good catch

Reply:
Ah, actually this one is a timestamp type (ref: https://github.com/18F/identity-idp/blob/f02e8cc650ba520a0933bfa150c8b45ac68d81f8/lib/identity_config.rb#L232)

Reply:
oh whoops, I think @aduth's comment better addresses the issue I thought I saw: https://github.com/18F/identity-idp/pull/5238#discussion_r677564241